var/spack/repos/builtin/packages/exago/package.py

```diff
@@ -46,7 +46,19 @@ class Exago(CMakePackage, CudaPackage, ROCmPackage):
 
     conflicts("~hiop~ipopt", msg="ExaGO needs at least one solver enabled")
 
-    # Dependencides
+    # release and debug flags
+    variant(
+        "release",
+        default=False,
+        description="Enable release type for packages exago, hiop, raja, magma, camp",
+    )
+    variant(
+        "debug",
+        default=False,
+        description="Enable RelWithDebInfo for packages exago, hiop, raja, magma, camp",
+    )
```

Comment:
Hmm, I kind of disagree with this, since `+release` is a proxy for `build_type=Release` and `+debug` is a proxy for `build_type=RelWithDebInfo`. Both options are already covered.

Reply:
We use this variant to also control the build type of dependency packages - are you suggesting we should change the name, or is this something we should do at all?

I'm satisfied if we should just handle this in our `spack.yaml` environment, but I think we have good intentions with the variant.